### ISSUE TITLE
Out of range: Read less bytes than requested

### ISSUE THREAD

**Issue by dongdu3** [maxorange/pix2vox#2]
Hello,

This work is very interesting! However, when I run the code ("python application.py"), it gets some errors like "Out of range: Read less bytes than requested", caused by "model = sgan.Model("params/sgan_model.ckpt")". Is the params model given right?

Thanks.

**Comment by maxorange** [comment]
Hi,

The model path is right. Are there three files (```sgan_model.ckpt.data-00000-of-00001```, ```sgan_model.ckpt.index``` and ```sgan_model.ckpt.meta```) in params folder?

p.s. I am using TensorFlow version 1.1.0.

**Comment by dongdu3** [comment]
Hi,

Thank you for your kind reply @maxorange . I have solved it and run the demo. The problem is that I didn't install the git-lfs correctly. So cannot git clone the correct scan_model.ckpt.data-00000-of-00001 model.

By the way, I also implemented the sketch-based modeling like yours, using 3D-VAE-GAN in the paper "Learning a Probabilistic Latent Space of Object Shapes via 3D Generative-Adversarial Modeling", but my latent space z seems no semantic meaning like yours, which displayed in the demo. That looks so cool!
